.gitignore

```diff
@@ -8,3 +8,9 @@ classes
 *.iml
 *.sc
 *.swp
+
+# Metals!
+.metals/
+.bloop/
+.vscode/
+project/
```

Comment:
I don't think we should ignore `project/`, right?

Reply:
Agreed :) a messy copy and paste